src/jinacld_tools/aws/services/s3.py

```diff
@@ -0,0 +1,37 @@
+import glob
+from pathlib import Path
+
+from jinacld_tools.aws.logger import get_logger
+from jinacld_tools.aws.client import AWSClientWrapper
+
+
+class S3:
+    """Wrapper around boto3 to upload to/download from S3 bucket
+    """
+
+    def __init__(self, bucket: str):
+        self._client = AWSClientWrapper(service='s3').client
+        self._bucket = bucket
```

Comment:
I guess its debatable if this should be part of the object or just a param in add/get methods, but I dont have an opinion here really.

Reply:
I would argue that since this thing is called S3 the user can expect that one instance is coupled to a bucket.
But not a strong opinion either. Maybe `S3Bucket` is even more explicit? And `bucket_name` instead of `bucket` for the constructor?

Reply:
Refactored in https://github.com/jina-ai/cloud-helper/pull/3/commits/e88b3fe3b50d9fabb5b05553edb5628dfa187647